sdk/core/System.ServiceModel.Rest/src/Temporary/IUtf8JsonWriteable.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+
+namespace System.ServiceModel.Rest.Shared.Core.Serialization
+{
+    /// <summary>
+    /// TBD.
+    /// </summary>
+    public interface IUtf8JsonWriteable
```

Comment:
This will go away when @m-nash's IModelSerializable and IModelJsonSerializeable types land.